README.md

```diff
@@ -62,6 +62,14 @@ tcld apikey create --name <api-key-name> --description <api-key-description> --d
 ```
 tcld apikey list
 ```
+
+### List API keys for a specific owner (service account or user):
+Note: Only Global Admins may list API keys of other users/service
```

Comment:
looks good! how about simplifying to `Note: only Global Admins may list API keys of other users and service accounts.`?